### PR TITLE
Add .nvmrc file

### LINKED ISSUE #237: Uglify silently failing with latest version of Node
### Describe the bug:
When building the extension Uglify fails to create the minified version of `assets/js/admin-ga-settings.js` without reporting an error. This was only noticed due to the SVN diff highlighting the file was missing when submitting to the .org repo.

### Steps to reproduce:
1. Clone repo
2. Run `nvm use node` to get the latest version
3. Run `npm i`
4. Run `npm run uglify`

### Expected behavior:
The file `assets/js/admin-ga-settings.min.js` should be created once the process is complete.

### Actual behavior:
There is no feedback suggesting it failed but the file isn't created.

### Additional details
We could look to update the version of Uglify but should also add a `.nvmrc` file to avoid future issues.

### PR DESCRIPTION
**Pull request by martynmjones**
Closes #237

Adds a `.nvmrc` file specifying the latest version of Node that supports `$npm_package_*` to ensure our build script is successful.

### Checks:
* [x] Does your code follow the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards?
* [x] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Detailed test instructions:

1. `nvm use` before building the extension
2. `npm i`
3. `npm run uglify`
4. Confirm that `assets/js/admin-ga-settings.min.js` was successfully created

### Changelog entry

> Add - .nvmrc file

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v14
```
Comment (tomalec) [inline .nvmrc]: :nail_care: I'd add a line break at the end to match our styles

**Review by tomalec — APPROVED** [review]
Tested locally and checked the code. LGTM, thanks :)

**Comment by tomalec** [comment]
Also, :nail_care: , I'd mark it as `Dev` change in the changelog, but add is good to me as well.

**Comment by tomalec** [comment]
@martynmjones FWIW, changing GitHub label would only affect GitHub release notes, Woorelease would pick up what's in `### Changelog entry` section, so still
> Add -

**Comment by martynmjones** [comment]
> @martynmjones FWIW, changing GitHub label would only affect GitHub release notes, Woorelease would pick up what's in `### Changelog entry` section, so still

Ah I see what you mean! Thanks for highlighting @tomalec  - I'll keep it in mind going forward 👍

**Pull request merged**